Source/Relativity.Test.Helpers/Relativity.Test.Helpers.Example.NUnit/app.config

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <configSections>
+    <section name="kCura.Config" type="System.Configuration.DictionarySectionHandler, System, Version=1.0.3300.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" />
+  </configSections>
+  <kCura.Config configSource="kCuraConnection.config" />
+  <appSettings file="appSettings.config">

```

Comment:
Even though these are generic values should not be checked in.  Should be using examples only!